addon/components/file-upload.hbs

```diff
@@ -0,0 +1,26 @@
+<label
```

Comment:
I think we should add `...attributes` here as well? I'm guessing people might have been using `@class` for additional styling. If we add `...attributes` they can switch to `class`.

Reply:
Ah.. yes 😅  I'm a bit conflicted about what to do here. This has been discussed a bit already in #293 and #316.

Ideally this PR paves the way for a full breaking-change rebuild of the public API.

I thought I'd pay all the tech debt while we're deciding the specifics, and then we can release that as a minor much more quickly. It means we won't be blocking anyone upgrading to Ember 4 either.

@jelhan is kindly going to review this soon too, I'll wait to hear his thoughts.

Reply:
Woops, sorry! I haven't seen those discussions. Just thought I'd mention it just in case.

Reply:
On balance I've reconsidered this and added `...attributes` to the <label> tag